snforge_std/src/cheatcodes/contract_class.cairo

```diff
@@ -2,21 +2,6 @@ use starknet::{ContractAddress, ClassHash, testing::cheatcode, SyscallResult};
 use super::super::byte_array::byte_array_as_felt_array;
 use core::traits::Into;
 
-#[derive(Drop, Clone)]
-struct RevertedTransaction {
-    panic_data: Array::<felt252>,
-}
-
-trait RevertedTransactionTrait {
-    fn first(self: @RevertedTransaction) -> felt252;
-}
```

Comment:
Wasn't this documented? I don't see any deletions of that from the docs

Reply:
Not separately as a trait. Only mentioned in the return types.